### DISCUSSION TITLE
Support for System Dark Mode (automatic Switch)

### DISCUSSION THREAD

**Discussion by AJK-Muc** [apache/netbeans#5540] [Q&A] [answered]
Since I really enjoy on all my devices (workstation, tablet, phone) automatic switching from dark to light (in sync with the daylight), it annoys me a little bit that I have to restart Netbeans twice every day while changing the theme from dark to light on sunrise and again from light to dark on sunset.

Q1: Is there any easy way to tell Netbeans to simply use the users system settings (like automatic dark mode)?
Q2: Is it possible to create themes with media queries like CSS `@media (prefers-color-scheme: dark) {}`

**Comment by mbien** [comment, marked as answer]
we have a draft PR open which might make it possible to change the theme without a restart, see #3523. There is however no guarantee that this will make it in or how well this will actually work in practice.

and thanks to @neilcsmith-net you can indeed adjust the theme manually (via properties and no media queries), see #4719

**Reply by AJK-Muc** [reply]
thanks a lot for that helpful information 😊
